src/App.tsx

```diff
@@ -2,6 +2,9 @@ import './App.scss';
 
 import { NavigationBar, Signup } from 'components';
 import { Input } from './components/inputs/Input';
+import { default as Button } from 'components/Button';
+import { ButtonTheme } from 'components/Themes/CustomButtonTheme'
+import { ThemeProvider } from '@mui/material/styles';
```

Comment:
Try to keep order in the imports
First, the imports from node_modules
blank line
local file imports 

also import Input and Button like NavigationBar, Signup

Reply:
done